medusa/server/web/config/post_processing.py

```diff
@@ -213,19 +220,13 @@ def isNamingValid(pattern=None, multi=None, abd=False, sports=False, anime_type=
             return 'invalid'
 
     @staticmethod
-    def isRarSupported():
-        """
-        Test Packing Support:
-            - Simulating in memory rar extraction on test.rar file
-        """
-
+    def isRarSupported(unrar_tool):
```

Comment:
No, it's changing one route, which doesn't follow our current convention. And you know what happens after that.. nothing! As nobody will feel him/her self called to change the rest.